versions.json

```diff
@@ -1 +1 @@
-["8.2", "8.1", "8.0", "1.3", "1.2", "1.1", "1.0", "0.26", "0.25"]
```

Comment:
Associated with this change would be the removal of all archived docs from this branch.